pages/options/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { createRoot } from 'react-dom/client';
 import '@src/index.css';
-import '@extension/ui/dist/global.css';
+import '@extension/ui/lib/global.css';
```

Comment:
Is this an intentional change?

Reply:
Yeah, because it have no sense, to put `global.css` inside `dist` if it will be bundle with this file